### PR TITLE
Created release v0.1.2

### PR DESCRIPTION
**Pull request by craigdbarber**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 				 
  ### Fixed
 
+## [0.1.2] - 2020-02-28
+
+ ### Security
+ 
+ ### Added
+ - Added support for setting up APPLICATION DEVELOPER CREDENTIALS
```
Comment (averikitsch) [inline CHANGELOG.md]:
```suggestion
 - Added support for setting up GOOGLE APPLICATION CREDENTIALS
```

**Comment by craigdbarber** [comment]
Correct, each action has its own separate CHANGELOG.md file.

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 				 
  ### Fixed
 
+## [0.1.2] - 2020-02-28
+
+ ### Security
+ 
+ ### Added
+ - Added support for setting up GOOGLE APPLICATION CREDENTIALS
```
Comment (sethvargo) [inline CHANGELOG.md]: Can specify creds as b64 or plain json

Comment (craigdbarber) [reply CHANGELOG.md]: Done.

**Pull request merged**